spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -1129,14 +1129,14 @@
             message = strip_tags(t('doc_auth.errors.selfie_not_live_or_poor_quality_heading'))
             expect(page).to have_content(message)
             detail_message = strip_tags(t('doc_auth.errors.alerts.selfie_not_live_or_poor_quality'))
-            security_message = strip_tags(
+            warning_message = strip_tags(
               t(
                 'idv.failure.attempts_html',
                 count: IdentityConfig.store.doc_auth_max_attempts - 1,
               ),
             )
 
-            expect(page).to have_content(detail_message << "\n" << security_message)
+            expect(page).to have_content(detail_message << "\n" << warning_message)
```

Comment:
while we're here, I would recommend changing to a way that doesn't mutate the value
```suggestion
            expect(page).to have_content("#{detail_message}\n#{warning_message}")
```